xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp

```diff
@@ -846,6 +848,14 @@ void CDVDVideoCodecAndroidMediaCodec::SetDropState(bool bDrop)
     m_videobuffer.iFlags &= ~DVP_FLAG_DROPPED;
 }
 
+void CDVDVideoCodecAndroidMediaCodec::SetCodecControl(int flags)
+{
+  if (m_codecControlFlags != flags)
+    if (g_advancedSettings.CanLogComponent(LOGVIDEO))
```

Comment:
That's a c/p from rpi, but yeah, not great now that I look at it, but working @popcornmix